src/components/NavBar.css

```diff
@@ -0,0 +1,73 @@
+.navbar {
```

Comment:
Please put in folder structure:

For coupling
> components
>> NavBar
>>> NavBar.js
>>> NavBar.css

Reply:
I will fix it and push code again.